### ISSUE TITLE
Role dependencies don't work with ansible-galaxy

### ISSUE THREAD

**Issue by HanXHX** [ceph/ceph-ansible#388]
Hi,

When you install roles from ansible-galaxy, meta/main.yml expects dependencies as `user`.`role_name`. However, all roles uses "local" fomula. It breaks playbooks.

Example in https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-mds/meta/main.yml
```
dependencies:
  - { role: ceph-common, when: not docker }
```

Expected:
```
dependencies:
  - { role: leseb.ceph-common, when: not docker }
```

Cheers!

Emilien

**Comment by leseb** [comment]
Looking into that.

**Comment by HanXHX** [comment]
I think you can:

- provides new ansible.cfg (called by Vagrant)
- force use roles from your git repo
- rename

In my work, I use this dirty script to work with galaxy:

```
#!/bin/bash                                                                                                                                                                                   

owner="leseb"
roles="ceph-common ceph-mds ceph-mon ceph-osd ceph-restapi ceph-rgw"
role_dir="$PWD/roles"

[ ! -d $role_dir ] && mkdir $role_dir 

rm -fr "$role_dir/*"

for r in $roles
do
    ansible-galaxy install -f leseb.$r -p $role_dir 
done

rename 's/leseb\.//' roles/*
```

**Comment by leseb** [comment]
I'm working on this, will push a fix soon.

**Comment by leseb** [comment]
Fixed by #591
